### DISCUSSION TITLE
Tailscale setup  | (...) port 443: Name does not resolve => error in logs but everything works?

### DISCUSSION THREAD

**Discussion by gromoslaw-kroczka** [nextcloud/all-in-one#3713] [Questions] [answered]
### Steps to reproduce
1. Set up Nextcloud AIO with Tailscale subdomain
2. Everything seems to work fine
3. You've got below log every 5 sec in nextcloud-aio-nextcloud container:
`nc: getaddrinfo for host "nextcloud.<tailnet>.ts.net" port 443: Name does not resolve`

### Expected behavior 
When everything works then it shouldn't be any error log message?

### Actual behavior 
There are log errors when everything seems to work well.

### Host OS 
Debian 12 on Proxmox VM

#### Nextcloud AIO version 
Nextcloud AIO v7.5.1

#### Current channel 
latest

#### Other valuable info 
Tailscale container running alongside Nextcloud AIO

```docker-compose
services:
  nextcloud:
    container_name: nextcloud-aio-mastercontainer
    image: nextcloud/all-in-one:latest
    restart: unless-stopped
    networks:
      - nextcloud-aio
    ports:
      - 8086:8080
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
    environment:
      - APACHE_PORT=11000
      - APACHE_IP_BINDING=0.0.0.0
      - SKIP_DOMAIN_VALIDATION=true

  nextcloud-tailscale:
    container_name: nextcloud-tailscale
    image: tailscale/tailscale:latest
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
      - NET_RAW
    network_mode: host
    hostname: nextcloud
    volumes:
      - /dev/net/tun:/dev/net/tun
      - /home/<user>/docker/nextcloud/tailscale/varlib:/var/lib
      - /home/<user>/docker/nextcloud/tailscale/tmp:/tmp
    environment:
      - TS_STATE_DIR=/var/lib/tailscale

networks:
  nextcloud-aio:
    name: nextcloud-aio

volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer

````

**Comment by szaimen** [comment, marked as answer]
Hi, apparently the container cannot resolve the tailscale domain. Unfortunatley I cannot help further as I dont have such a setup and also cannot reproduce it with a "normal" setup without talescale. So for help better refer to https://help.nextcloud.com

**Comment by gromoslaw-kroczka** [comment]
I've noticed few more details:
1) Along with errors in containers logs there are a lot of DNS request to my nextcloud domain (every 5 sec)
2) Restart of the whole container fixes the issue (both errors and DNS requests)
3) Both sypmtoms reoccur (begin) each night after Nextcloud Borg backup (internal Nextcloud)
